docs/content/docs/reference/react/hooks.mdx

```diff
@@ -0,0 +1,393 @@
+---
+title: 'Hooks'
+---
+
+## Authentication Hooks
+
+### Email + password registration
+
+```js
+const { signUp, isLoading, isSuccess, needsVerification, isError, error } =
+  useEmailPasswordSignUp(email?: string, password?: string, options?: SignUpOptions )
+```
+
+- `signUp: (email?: string, password?: string) => void` triggers the user registration. The email/password arguments will take precedence over the possible state values used when creating the hook.
```

Comment:
"triggers the user registration" => "is used to sign up a user" or something similar. Effectively changing "registration" to "sign up".